docs/user-documentation/getting-started.md

```diff
@@ -44,12 +44,13 @@ After that you need to import the Bee class and create a bee instance connecting
 Be aware, if you will pass invalid URL the constructor will throw an exception!
```

Comment:
Modify the part before this to explain `Bee` and `BeeDebug`?

Reply:
Yeah that would great addition!